website/docs/components/dropdown/partials/code/how-to-use.md

```diff
@@ -126,8 +126,10 @@ In contexts where the Dropdown needs to be _inline_, to inherit the alignment fr
 
 You can explicitly control the height or width of a list. Any acceptable value (px, rem, em) can be declared:
 
+Note: The `@height` argument actually sets a `max-height` which prevents the list from growing past a certain height.
```

Comment:
[non-blocking] This could be but into a `Info` banner similarly to how we style other types of "notes"

Reply:
Thanks for the feedback. I can turn it into an Info banner.